src/main/gui/create.ts

```diff
@@ -67,13 +67,25 @@ export const createGuiApp = (args: OpenArguments) => {
     // Some APIs can only be used after this event occurs.
     app.on('ready', createWindow);
 
+    // This is to make sure that on macOS the app is quit when the window is closed
+    app.on('window-all-closed', () => {
+        if (process.platform === 'darwin') {
+            app.quit();
+        }
+    });
```

Comment:
This isn't necessary anymore after #2051.

Reply:
Perfect. Going to close this PR.